js/main.js

```diff
@@ -22,3 +27,51 @@ navLinks.forEach((navLink) => {
     activate();
   });
 });
+
+// form validity check
+
+function setSuccess(input) {
+  const inputWrapper = input.parentElement;
+  inputWrapper.classList.add('success');
+}
+
+function setError(input, message) {
+  const inputWrapper = input.parentElement;
+  const small = inputWrapper.querySelector('small');
+  inputWrapper.classList.add('error');
+  small.innerText = message;
+}
+
+function validateInput() {
+  const emailValue = email.value.trim();
+  const fnameValue = fname.value.trim();
+  const messageValue = message.value.trim();
+  let isFalse = false;
+
+  if (/^[a-z0-9._-]+@[a-z0-9.-]+\.[a-z]{2,4}$/g.test(emailValue)) {
+    setSuccess(email);
+    isFalse = true;
+  }
+  setError(email, 'Invalid email');
```

Comment:
- [ ] `required`
- [ ] Good so far! 👏 
- [ ]  as the stated in the requirements.
- [ ]  If the validation is not OK, you show an error message to the user `near the submit button` informing them of the error and the form is not sent.
-  kindly move the error message near the submit button. 🥂